src/tutorial/microbe_editor/NegativeAtpBalanceTutorial.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Tutorial
+{
+    using System;
+
+    /// <summary>
+    ///   Tells the player about negative ATP balance
+    /// </summary>
+    public class NegativeAtpBalanceTutorial : TutorialPhase
+    {
+        public override string ClosedByName => "NegativeAtpBalanceTutorial";
+
+        public override void ApplyGUIState(MicrobeEditorTutorialGUI gui)
+        {
+            gui.NegativeAtpBalanceTutorialVisible = ShownCurrently;
+        }
+
+        public override bool CheckEvent(TutorialState overallState, TutorialEventType eventType, EventArgs args,
+            object sender)
+        {
+            switch (eventType)
+            {
+                case TutorialEventType.MicrobeEditorPlayerEnergyBalanceChanged:
+                    var energyBalanceEventArgs = args as EnergyBalanceEventArgs;
+                    var energyBalanceInfo = energyBalanceEventArgs?.EnergyBalanceInfo;
+                    bool isNegativeAtpBalance =
+                        energyBalanceInfo?.TotalProduction < energyBalanceInfo?.TotalConsumption;
+
+                    if (!HasBeenShown && isNegativeAtpBalance && CanTrigger && !overallState.TutorialActive())
```

Comment:
This is more of a note for myself, but I'll need to playtest after the event trigger location change to make sure it doesn't take priority from the intended first set of tutorials in the editor.